### ISSUE TITLE
Training on datasets other than MNIST

### ISSUE THREAD

**Issue by drewgillson** [XifengGuo/CapsNet-Keras#41]
I have 26 classes contained in a dataset with the same pixel dimensions as MNIST and I have 1,000 samples for each class. I've successfully trained a CapsNet that can classify around 12-14 of my 26 classes with 99% accuracy, but any more classes than this and the loss will converge on a single high value and never improve. There seems to be a specific cut-off point where the network architecture fails above a certain number of classes.

I've experimented with increasing the values for the number of dimensions in the PrimaryCaps layer, setting different values for the learning rate, and changing the batch size, but I haven't been able to solve the issue.

Do you have any tips for other things I should be trying Xifeng? (I'm training on AWS p2.xlarge Tesla K80 GPU)

Thanks!

**Comment by XifengGuo** [comment]
@drewgillson I haven't encountered this problem. What’s the highest acc before it stops improving?

**Comment by drewgillson** [comment]
I ended up taking a different approach and cannot remember the details, will close the issue.
